view/src/main/java/cmm/apps/esmorga/view/eventdetails/model/EventDetailsUiState.kt

```diff
@@ -10,11 +13,29 @@ data class EventDetailsUiState(
     val locationLat: Double? = null,
     val locationLng: Double? = null,
     val navigateButton: Boolean = locationLat != null && locationLng != null,
+    val userJoined: Boolean = false,
+    val isAuthenticated: Boolean = false,
```

Comment:
you do not need to have these `userJoined` & `isAuthenticated` values in the view, they are not used anywhere.